src/API/Helpers/ApiClient.php

```diff
@@ -36,67 +65,81 @@ public static function getInfoHeadersData()
 
+    /**
+     * API domain.
+     *
+     * @var string
+     */
     protected $domain;
 
+    /**
+     * Base API path.
+     *
+     * @var string
+     */
     protected $basePath;
 
+    /**
+     * Headers to set for all calls.
+     *
+     * @var array|mixed
+     */
     protected $headers;
 
+    /**
+     * Options to pass to the Guzzle HTTP library.
+     *
+     * @var array
+     */
     protected $guzzleOptions;
 
+    /**
+     * Data type to return from the call.
+     * Can be "body" (default), "headers", or "object".
+     *
+     * @var string|null
+     *
+     * @see \Auth0\SDK\API\Helpers\RequestBuilder::call()
+     */
     protected $returnType;
 
-    public function __construct($config)
+    /**
+     * ApiClient constructor.
+     *
+     * @param array $config Configuration for this client.
+     */
+    public function __construct(array $config)
     {
         $this->basePath      = $config['basePath'];
         $this->domain        = $config['domain'];
-        $this->returnType    = isset( $config['returnType'] ) ? $config['returnType'] : null;
-        $this->headers       = isset($config['headers']) ? $config['headers'] : [];
-        $this->guzzleOptions = isset($config['guzzleOptions']) ? $config['guzzleOptions'] : [];
+        $this->returnType    = $config['returnType'] ?? null;
+        $this->headers       = $config['headers'] ?? [];
+        $this->guzzleOptions = $config['guzzleOptions'] ?? [];
 
         if (self::$infoHeadersDataEnabled) {
             $this->headers[] = new Telemetry(self::getInfoHeadersData()->build());
         }
     }
 
-    /**
-     * Magic method to map HTTP verbs to request types.
-     *
-     * @deprecated 5.6.0, use $this->method().
-     *
-     * @param string $name      - Method name used to call the magic method.
-     * @param array  $arguments - Arguments used in the magic method call.
-     *
-     * @return RequestBuilder
-     */
-    public function __call($name, $arguments)
-    {
-        $builder = new RequestBuilder([
-            'domain' => $this->domain,
-            'basePath' => $this->basePath,
-            'method' => $name,
-            'guzzleOptions' => $this->guzzleOptions,
-            'returnType' => $this->returnType,
-        ]);
-
-        return $builder->withHeaders($this->headers);
-    }
-
     /**
      * Create a new RequestBuilder.
-     * Similar to the above but does not use a magic method.
      *
-     * @param string  $method           - HTTP method to use (GET, POST, PATCH, etc).
-     * @param boolean $set_content_type - Automatically set a content-type header.
+     * @param string  $method           HTTP method to use (GET, POST, PATCH, etc).
+     * @param boolean $set_content_type Automatically set a content-type header.
      *
      * @return RequestBuilder
      */
-    public function method($method, $set_content_type = true)
+    public function method(string $method, bool $set_content_type = true) : RequestBuilder
```

Comment:
Why is there a space before `: return-type` ? PSR-12 states the form as `function name(): return-type`.